app/lang/pl/tb.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+return [
+    'added_new_plant' => 'Dodałem nową roślinę: <a href="{url}">{name}</a>',
```

Comment:
In this section english version:
"I added", "I moved" sounds asexually but in polish I need to specify it like
"Przeniosłem" - male
"Przeniosłam" - female

Other option is to use impersonal form like
"Dodano nową roślinę" -> "New plan was added"